### DISCUSSION TITLE
Structure of the three features in regards to the main page?

### DISCUSSION THREAD

**Discussion by TafeidMatt** [Kade03/Tafe22S203#2] [General]
How are we going to add our features to the main page?

My thoughts, we just develop on the main page itself, I will use the top left, someone use bottom left and someone else use the bottom right.
No need for extra pages, just coded all on the main page.

e.g. math calc at the top left: I will add 2 buttons and 2 text boxes. 
textbox one is input number, button one is addition button,
textbox two is output of equation, button two is equals button.
![image](https://user-images.githubusercontent.com/112943875/189530209-69d1307d-3a8e-426d-981e-144c48eb46b0.png)

**Comment by Kade03** [comment]
not 100% on the idea as the program would look a lot smoother with different pages for each feature. however it is a good alternative if its too difficult to code the buttons to open the feature

**Reply by TafeidMatt** [reply]
It would be better if we all had the same design.

**Comment by Kade03** [comment]
if you create the button click event for your feature use this code so that your button works correctly and changes to your feature page. that way we're not adding our features to the main page and making a messy project 

private void FeatureName(object sender, RoutedEventArgs e)
		{
			this.Frame.Navigate(typeof(FeatureName));

		}

**Comment by WillLeslie03** [comment]
![menu bar](https://user-images.githubusercontent.com/112096335/189812653-40ebffcb-fd24-4806-88bd-189b3865c3a4.png)
what if we did it like a menu bar like this

**Reply by Kade03** [reply]
do you mean as the buttons for the project?
